README.md

```diff
@@ -78,13 +88,15 @@ If you want to limit the available views, you can import each of the views direc
 This might be useful when overriding templates and defining permissions.
 
 ### Using out of the box templates
+
 The package comes with some templates to allows uses to manage and complete forms. If you wish to use these templates, you will need to download `django-crispy-forms` and update your `settings.py` file. Follow the instructions below:
 
-```
+```python
```

Comment:
It'd change this to "```bash" as strictly speaking it's not a python snippet as it's something we'd run in shell.

Reply:
Idk how I missed that, resolved on 7b9462fdabdf38e926bb4963649210fa873bd570